app/javascript/api/adapters/listing/RentalListingAdapter.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Listing,
+  ListingStatus,
+  CSVFormattingType,
+  CountyCode,
+} from "@bloom-housing/backend-core/types"
+
+import RailsRentalListing from "../../types/rails/listings/RailsRentalListing"
+import { Adapter } from "../adapter"
+import ListingPropertyAdapter from "./ListingPropertyAdapter"
+
+const toDate = (s: string): Date => new Date(s) // todo: handle different formats, actually implement this properly
+
+const ListingAdapter: Adapter<RailsRentalListing, Listing> = (item: RailsRentalListing) => ({
+  status: ListingStatus.active,
+  urlSlug: "", // todo: populate this field
+  displayWaitlistSize: false,
+  CSVFormattingType: CSVFormattingType.basic,
+  countyCode: CountyCode.Alameda, // todo: Add san francisco county to Bloom's enum
+  showWaitlist: item.hasWaitlist,
+  preferences: [],
+  property: ListingPropertyAdapter(item),
+  applicationAddress: null,
+  applicationPickUpAddress: null,
+  leasingAgentAddress: null,
+  leasingAgents: null,
+  id: item.listingID,
+  createdAt: null, // todo: update this field
+  updatedAt: null, // todo: update this field
+  applicationMethods: [],
+  assets: [
+    {
+      id: "",
+      createdAt: null,
+      updatedAt: null,
+      label: "building",
+      fileId: item.imageURL,
```

Comment:
something funky is going on here where on full, the imageURL attribute is a cloudfront url, and here it's coming through as a MOHCD image url. I wonder if there's some processing going on in the frontend that we're missing.

Reply:
actually after a bit of googling I think it's just because it's a review app and it doesn't have all of the cloudfront urls cached in a local db, so this should be ok! 

Code where that's done here:
 https://github.com/SFDigitalServices/sf-dahlia-web/blob/main/app/services/force/listing_service.rb#L107-L118